modules/nodejs/manifests/init.pp

```diff
@@ -9,15 +9,9 @@
 #   Default: `undef`
 #
 class nodejs(
-  $version = undef,
+  $version = 'present',
```

Comment:
The code below needs to be changed to `$version` rather than `$ensure`

Reply:
Nice catch. Fixed and pushed. Functional testing would catch this ;)